RibonDAO/interface
# Feat/prod3 5

# Description

- Adds incorrect email error treatment

## Does this pull request close any open issues?

- [#prod3-5](https://ribon.atlassian.net/browse/PROD3-5?atlOrigin=eyJpIjoiMjU4MTE3ZjJmYjgyNDQ3NjkyMjU5NTM0YzdiNTc5NjUiLCJwIjoiaiJ9)

## How to test

- "log in" with email
- try to sign in with a different email on google and apple (sign in extra ticket page)
- see if the "wrong email" modal appears
